src/app/components/Tab/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Tab } from "@headlessui/react";
+import { classNames } from "~/app/utils/index";
+
+type Props = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  active: boolean;
+  icon?: React.ReactNode;
+  leftTab?: boolean;
+  rightTab?: boolean;
+  label?: string;
+};
+
+export default function TabComponent({
```

Comment:
I would call this component "Tab" and rename the imported one 
`import { Tab as HeadlessUiTab } from ...`